README.md

```diff
@@ -74,21 +74,29 @@ Usually the message on the DONE queue will be processed by the captionWF in comm
 docker run --rm --tty --env-file .env sul-speech-to-text --receive
 ```
 
-## The Job File
+## The Job Message Structure
 
-The job file is a JSON object that contains information about how to run Whisper. Minimally it contains the Job ID, which will be used to locate media files in S3 that need to be processed. 
+The job is a JSON object (used as an SQS message payload) that contains information about how to run Whisper. Minimally it contains the Job ID and a list of file names, which will be used to locate media files in S3 that need to be processed.
 
 ```json
 {
-  "id": "8EB51B59-BDFF-4507-B1AA-0DE91ACA388F",
+  "id": "gy983cn1444-v2",
+  "media": [
+    "snl_tomlin_phone_company.mp4"
+  ],
```

Comment:
might be jumping the gun here if you have a change in the works where the job path in the bucket just gets listed, and no explicit file list is needed.  but seems like the current state of the code on `main` still needs this list?

Reply:
we could make the media include the full key in the bucket, e.g. `gy983cn1444-v2/snl_tomlin_phone_company.mp4` instead of just `snl_tomlin_phone_company.mp4`

Reply:
or the container could do that since it has all of of the info (if we don't want to be explicit in the message itself)

Reply:
i don't think i have a preference, other than maybe a very slight bias towards what the code currently does (concatenate job id and media entry to get full key), just because that seems as good as the other options in principle, and it's already implemented, so feels like an acceptable path of least resistance.

Reply:
My latest PR sends the media element in the message with an array of filenames (with the job_id key part, so basically from the root of the bucket).  If we decide later this is redundant and not needed, we can remove it from the message, but it also won't do any harm if its ignored.

Reply:
d'oh, i think we had discussed this last week, but my fault if i forgot that we decided to just list the bucket and not send a list of file names explicitly in the job message.  i'm totally fine with that approach.  i did put up this PR after i put #20 in the ready column 🤦, so i suspect this was just me forgetting what we'd all discussed and settled on at the end of last week.

i'll amend this PR to get rid of the file list stuff.

sorry @peetucket for the misguided feedback i left on your PR yesterday!

Reply:
leaving this in since we finally settled on closing #20